pages/public_cloud/ai_machine_learning/training_tuto_09_train_rock-paper-scissors/guide.en-gb.md

```diff
@@ -0,0 +1,263 @@
+    User Volume Id: bf7b5c4b-c61e-4d17-b6d8-248de56834b6
+```
+
+You can access to the execution logs of your job with the CLI:
+
+```bash
+ovhai job logs <job id> -f       
+```
+
+The output should be similar to this:
+
+```bash
+$ ovhai job logs c0c0878c-5564-4660-889a-65724f6e3056 -f
+
+Job is running
+2023-10-06T15:23:14Z [job] Ultralytics YOLOv8.0.194 🚀 Python-3.8.18 torch-2.1.0+cu121 CUDA:0 (Tesla V100S-PCIE-32GB, 32501MiB)
+2023-10-06T15:23:14Z [job] Setup complete ✅ (60 CPUs, 172.7 GB RAM, 1191.4/3519.5 GB disk)
+2023-10-06T15:23:14Z [job] Downloading https://github.com/ultralytics/assets/releases/download/v0.0.0/yolov8n.pt to 'yolov8n.pt'...
+100%|██████████| 6.23M/6.23M [00:00<00:00, 170MB/s]
+2023-10-06T15:23:15Z [job] Ultralytics YOLOv8.0.194 🚀 Python-3.8.18 torch-2.1.0+cu121 CUDA:0 (Tesla V100S-PCIE-32GB, 32501MiB)
+2023-10-06T15:23:15Z [job] engine/trainer: task=detect, mode=train, model=yolov8n.pt, data=/workspace/data/rock-paper-scissors/data.yaml, epochs=50, patience=50, batch=16, imgsz=640, save=True, save_period=-1, cache=False, device=0, workers=8, project=None, name=None, exist_ok=False, pretrained=True, optimizer=auto, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_mosaic=10, resume=False, amp=True, fraction=1.0, profile=False, freeze=None, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, show=False, save_txt=False, save_conf=False, save_crop=False, show_labels=True, show_conf=True, vid_stride=1, stream_buffer=False, line_width=None, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_masks=False, boxes=True, format=torchscript, keras=False, optimize=False, int8=False, dynamic=False, simplify=False, opset=None, workspace=4, nms=False, lr0=0.01, lrf=0.01, momentum=0.937, weight_decay=0.0005, warmup_epochs=3.0, warmup_momentum=0.8, warmup_bias_lr=0.1, box=7.5, cls=0.5, dfl=1.5, pose=12.0, kobj=1.0, label_smoothing=0.0, nbs=64, hsv_h=0.015, hsv_s=0.7, hsv_v=0.4, degrees=0.0, translate=0.1, scale=0.5, shear=0.0, perspective=0.0, flipud=0.0, fliplr=0.5, mosaic=1.0, mixup=0.0, copy_paste=0.0, cfg=None, tracker=botsort.yaml, save_dir=runs/detect/train
+...
+```
+
+For more explanations about the CLI command for AI Training, please read this guide: [CLI Reference](/pages/public_cloud/ai_machine_learning/cli_15_commands_reference).
+
+Once you have your model ready, deploy the model to use it. This will be done with the AI Deploy tool.
+
+## Go further
+
+All the source code is available on the [OVHcloud GitHub organization](https://github.com/ovh/ai-training-examples/tree/main/jobs/audio/audio-classification).
+
+To create the application using the trained model, you can follow this tutorial: [Deploy an app for playing rock / paper / scissors](/pages/public_cloud/ai_machine_learning/deploy_tuto_15_rock_paper_scissors).
```

Comment:
Please add following line : 

If you need training or technical assistance to implement our solutions, contact your sales representative or click on [this link](https://www.ovhcloud.com/en-gb/professional-services/) to get a quote and ask our Professional Services experts for a custom analysis of your project.

Reply:
done 02f380c163308acddedd17c2313cd4ee23e91c56